nflow-engine/src/main/java/io/nflow/engine/internal/executor/WorkflowDispatcher.java

```diff
@@ -80,6 +86,8 @@ public void run() {
           sleep(false);
         }
       }
+    } catch (IOException | ReflectiveOperationException e) {
```

Comment:
Either move the try-catch-logging-rethow around `workflowDefinitions.postProcessWorkflowDefinitions()` or remove catch statement altogether.

Currently if normal operation should throw IOExceptions this would catch and hide them.